DEVELOPMENT.md

```diff
@@ -146,6 +146,26 @@ Follow the
 [instructions](https://www.knative.dev/docs/serving/gke-assigning-static-ip-address/)
 if you need to set up static IP for Ingresses in the cluster.
 
+### Deploy cert-manager
+1. Deploy `cert-manager` CRDs
+   ```shell
+   kubectl apply -f ./third_party/cert-manager-0.6.1/cert-manager-crds.yaml
+   while [[ $(kubectl get crd certificates.certmanager.k8s.io -o jsonpath='{.status.conditions[?(@.type=="Established")].status}') != 'True' ]]; do
+     echo "Waiting on Cert-Manager CRDs"; sleep 1
+   done
+   ```
+
+1. Deploy `cert-manager`
+
+   **Note**: The auto TLS feature has not been landed in Knative. At this point, you can skip this step.
+
+   If you want to use the feature of automatically provisioning TLS for Knative services, you need to install
+   the full cert-manager.
+   ```shell
+   # For kubernetes version 1.13 or above, --validate=false is not needed.
+   kubectl apply -f ./third_party/cert-manager-0.6.1/cert-manager.yaml --validate=false
```

Comment:
Why is this disabling validation?

Reply:
Looks like it's because cert-manager runs a job to configure this?

Reply:
Per the [installation doc](https://docs.cert-manager.io/en/latest/getting-started/install.html), this is related to validating the `caBundle` field of the `ValidatingWebhookConfiguration` resource. This [issue](https://github.com/kubernetes/kubernetes/issues/69590) was resolved in k8s 1.13.